# Bioinformatics training at the [Harvard Chan Bioinformatics Core](http://bioinformatics.sph.harvard.edu/)

## Training program description:

We provide bioinformatics training through both shorter workshops and longer in-depth courses. 

*No prior NGS or command line expertise is required for our workshops or courses unless explicitly stated.*

### Workshops

**Introduction to Next-Generation Sequencing (NGS) analysis**: The goal of these workshops (2-3 days) are to enable researchers to design their studies appropriately and perform prelimanary data analyses.

  * Introduction to R and ggplot2 (2 days)
  * Introduction to RNA-seq data analysis using High-Performance Computing (2 or 3 days)
  * Introduction to R and Differential Gene Expression (DGE) analysis (3 days)
  * Introduction to Differential Gene Expression (DGE) analysis (1.5 days)
  * Introduction to ChIP-seq using High-Performance Computing (3 days) (*In development*)

**Current topics in bioinformatics**: These short workshops (half-day or less) are designed to allow researchers, who have some familiarity with R or bash, to learn new tools and methods. 

  * Introduction to R & Visualizations with ggplot2 (*This is a Pre-requisite for the workshops listed below*)
  * Plotting and visualization in R using ggplot2 and other packages (*Prereq: familiarity with R*)
  * Functional analysis of gene lists (*Prereq: familiarity with R*)
  * Reproducible research using R: Report generation using Rmarkdown (*Prereq: familiarity with R*)
  * Differential Gene Expression (DGE) analysis with DESeq2 (*Prereq: familiarity with R*) (*In development*)

### In-Depth Next Generation Sequencing Analysis Courses

These intensive courses run for **8-12 days** and are aimed at bench biologists interested in learning how to independently perform NGS data analyses using best practices. Topics include:

  * High-Performance Computing (Orchestra)
  * NGS data analysis (RNA-Seq, ChIP-Seq, Variant calling)
  * Statistical analysis using R
  * Functional analysis with gene lists



