# Training program description:

The training team at the Harvard Chan Bioinformatics Core provides bioinformatics training through both shorter workshops and longer in-depth courses. Our current workshops and courses are designed to help biologists become comfortable with using tools to analyse high-throughput data. We are slowly beginning to expand this repertoire to include training for researchers with more advanced bioinformatics skills. See our current workshop schedule on our [training website](http://bioinformatics.sph.harvard.edu/training/).

*No prior NGS or command line expertise is required for our workshops or courses unless explicitly stated.*

### Introduction to Next-Generation Sequencing (NGS) analysis series: 

The goal of these workshops (2-3 days) are to enable researchers to design their NGS studies appropriately and perform preliminary data analyses.

| Training topic and link to lessons | Prerequisites | Workshop Duration |
:----------:|:----------:|:----------:|
| [R and ggplot2](https://hbctraining.github.io/Intro-to-R/) | None | 2 days |
| [RNA-seq data analysis using High-Performance Computing](https://hbctraining.github.io/Intro-to-rnaseq-hpc-O2/) | None | 2 - 3 days |
| [R and Differential Gene Expression (DGE) analysis](https://hbctraining.github.io/Intro-to-R-with-DGE/) | None | 3 days |
| [Differential Gene Expression (DGE) analysis](https://hbctraining.github.io/DGE_workshop/) | [R and ggplot2](https://hbctraining.github.io/Intro-to-R/) | 1.5 days |
| [ChIP-seq using High-Performance Computing](https://hbctraining.github.io/Intro-to-ChIPseq/) | None  | 3 days |
| Identifying variants in genome/exome sequencing data  | None | *In development* |

### Current topics in bioinformatics series:

These short workshops (half-day or less) are designed to allow researchers, who have some familiarity with R or bash, to learn new tools and methods. For the short workshops topics please [click here](https://hbctraining.github.io/Training-modules/).


### In-Depth Next Generation Sequencing Analysis Course

[This intensive course](https://hbctraining.github.io/In-depth-NGS-Data-Analysis-Course/) runs for between **8 to 12 days** and is aimed at bench biologists interested in learning how to **independently** perform NGS data analyses using *best practices*. Topics include:

  * High-performance computing
  * Best practice workflows for NGS data analysis (RNA-seq, ChIP-seq, Variant calling)
  * R for statistical analysis and data visualization
  * Functional analysis with gene lists
  * Additional skils and tools for better reproducibility like reports with RMarkdown, version control with Git/Github, etc.

## Contact us:

**Email:** [hbctraining@hsph.harvard.edu](mailto:hbctraining@hsph.harvard.edu)

**Webpage:** [http://bioinformatics.sph.harvard.edu](http://bioinformatics.sph.harvard.edu)

**Twitter:** [@bioinfocore](http://twitter.com/bioinfocore)
