# Training program description:

The training team at the Harvard Chan Bioinformatics Core provides bioinformatics training through both shorter workshops and longer in-depth courses. Our current workshops and courses are designed to help biologists become comfortable with using tools to analyse high-throughput data. We are slowly beginning to expand this repertoire to include training for researchers with more advanced bioinformatics skills.

*No prior NGS or command line expertise is required for our workshops or courses unless explicitly stated.*

### Workshops

**Introduction to Next-Generation Sequencing (NGS) analysis**: The goal of these workshops (2-3 days) are to enable researchers to design their NGS studies appropriately and perform preliminary data analyses.

  * [Introduction to R and ggplot2](https://hbctraining.github.io/Intro-to-R/) (2 days)
    
  * [Introduction to RNA-seq data analysis using High-Performance Computing](https://hbctraining.github.io/Intro-to-rnaseq-hpc-O2/) (2 - 3 days)
  
  * [Introduction to R and Differential Gene Expression (DGE) analysis](https://hbctraining.github.io/Intro-to-R-with-DGE/) (3 days)
  
  * [Introduction to Differential Gene Expression (DGE) analysis](https://hbctraining.github.io/DGE_workshop/) (1.5 days)
  
  * Introduction to ChIP-seq using High-Performance Computing (3 days) *(In development)*
  
  * Identifying variants in genome/exome sequencing data *(In development)*
  
**Current topics in bioinformatics**: These short workshops (half-day or less) are designed to allow researchers, who have some familiarity with R or bash, to learn new tools and methods. 

  * [Introduction to R & Visualizations with ggplot2](https://hbctraining.github.io/Training-modules/IntroR_ggplot2/)
  
  * [Plotting and visualization in R using ggplot2 and other packages](https://hbctraining.github.io/Training-modules/Visualization_in_R/) *(Prereq: familiarity with R)*
  
  * [Functional analysis of gene lists](https://hbctraining.github.io/Training-modules/DGE-functional-analysis/) *(Prereq: familiarity with R)*
  
  * [Reproducible research using R: Report generation using Rmarkdown](https://hbctraining.github.io/Training-modules/Rmarkdown/) *(Prereq: familiarity with R)*
  
  * Introduction to shell/bash *(In development)*
  
  * Intermediate shell/bash *(Prereq: familiarity with shell/bash*) *(In development)*
  
  * Version Control with Git *(Prereq: familiarity with HPC and bash*) *(In development)*
  
  * Introduction to [tidyverse R packages](https://www.tidyverse.org/packages/) for data visualization *(Prereq: familiarity with R*) *(In development)*
   
### In-Depth Next Generation Sequencing Analysis Course

[This intensive course](https://hbctraining.github.io/In-depth-NGS-Data-Analysis-Course/) runs for between **8 to 12 days** and is aimed at bench biologists interested in learning how to **independently** perform NGS data analyses using *best practices*. Topics include:

  * High-performance computing
  * Best practice workflows for NGS data analysis (RNA-seq, ChIP-seq, Variant calling)
  * R for statistical analysis and data visualization
  * Functional analysis with gene lists
  * Additional skils and tools for better reproducibility like reports with RMarkdown, version control with Git/Github, etc.

## Contact us:

**Email:** [hbctraining@hsph.harvard.edu](mailto:hbctraining@hsph.harvard.edu)

**Webpage:** [http://bioinformatics.sph.harvard.edu](http://bioinformatics.sph.harvard.edu)
